CIP-0008/README.md

```diff
@@ -1,13 +1,19 @@
 ---
 CIP: 8
 Title: Message Signing
-Authors: Sebastien Guillemot <sebastien@emurgo.io>
-Comments-URI:
-- https://github.com/Emurgo/EmIPs/pull/5
-- https://forum.cardano.org/t/message-signing-specification/41032
-- https://cardano.ideascale.com/a/dtd/Create-message-signing-standard/323158-48088
-Status: Draft
-Type: Standards
+Status: Active
+Category: Tools
+Authors:
+  - Sebastien Guillemot <sebastien@emurgo.io>
+Implementors:
+  - Mesh
+  - CardanoBallot
```

Comment:
```suggestion
  - Cardano Ballot <https://github.com/cardano-foundation/cf-cardano-ballot>
```

Reply:
or would the implementer be the CF? and the ballot is the specific implementatio n

Reply:
@Ryun1 I feel sure the Implementor should be linked through the software repo because there's far more detail on that GitHub page than there ever could be through the top level of the CF itself... and people will be able to file issues there if they have implementation questions or integration problems.